backend-diagram-converter/core/src/main/resources/message-templates.properties

```diff
@@ -114,6 +127,9 @@ delegate-implementation.severity=REVIEW
 script-job-type.message=Element '{{ elementLocalName }}' was transformed. Currently, script tasks are implemented like service tasks with job type '{{ jobType }}'. Please review your implementation.
 script-job-type.severity=REVIEW
 #
+result-variable-internal-script.message=Result variable is set to Zeebe script result variable. Please review.
+result-variable-internal-script.severity=REVIEW
```

Comment:
Could be INFO, as the variable name didn't change and is a fixed value.

Reply:
Fixed